web-common/src/runtime-client/watch-request-client.ts

```diff
@@ -33,17 +37,38 @@ export class WatchRequestClient<Res extends WatchResponse> {
   ) {}
 
   public start(): Unsubscriber {
```

Comment:
Feels to me like the `start()` method shouldn't additionally bake in logic to pause/cancel the operation. Could a better structure be to have a `pause()` or `cancel()` method that you can call on the `WatchRequestClient` when the page goes out-of-focus?

Reply:
It might be better to have everything in one place. The logic of calling all onReconnect should be here as well. Lets spend more time once the release is done.